toolkit/analysis_and_visualization/classification/assets/scripts/charts.js

```diff
@@ -0,0 +1,415 @@
+function sum(a, b) {
+    return a + b;
+}
+
+function drawChart() {
+
+    var chartData = google.visualization.arrayToDataTable([
+        ['  ', 'Match', 'Mismatch', 'No Label'],
+        ['Summary', data.stats.passCount, data.stats.totalMismatch, data.stats.totalNoGroundTruth]
+    ]);
+    var options = {
+        title: 'Overall Result Summary',
+        vAxis: {
+            title: 'Images'
+        },
+        width: 800,
+        height: 400
+    };
+    var chart = new google.charts.Bar(document.getElementById('Model_Stats'));
+    chart.draw(chartData, google.charts.Bar.convertOptions(options));
+}
+
+
+function drawTopKResultChart() {
+    var chartData = new google.visualization.DataTable();
+    chartData.addColumn('string', 'Top K');
+    chartData.addColumn('number', 'Matchs');
+    chartData.addRows([
+        ['Matched Top5 Choice', data.topCounts.reduce(sum, 0)],
+        ['MisMatched', data.stats.totalMismatch]
+    ]);
+    var options = {
+        title: 'Image Match/Mismatch Summary',
+        width: 750,
+        height: 400
+    };
+    var chart = new google.visualization.PieChart(document.getElementById('topK_result_chart_div'));
+    chart.draw(chartData, options);
+}
+
+
+function drawResultChart() {
+    var chartData = new google.visualization.DataTable();
+    chartData.addColumn('string', 'Top K');
+    chartData.addColumn('number', 'Matchs');
+    chartData.addRows([
+        ['Matched 1st Choice', data.topCounts[0]],
+        ['Matched 2nd Choice', data.topCounts[1]],
+        ['Matched 3rd Choice', data.topCounts[2]],
+        ['Matched 4th Choice', data.topCounts[3]],
```

Comment:
Codacy found an issue: ['data' is not defined.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)